code/modules/reqs/supplypacks.dm

```diff
@@ -807,7 +807,7 @@ CLOTHING
 /datum/supply_packs/clothing/combat_pack
 	name = "Combat Backpack"
 	contains = list(/obj/item/storage/backpack/lightpack)
-	cost = 20
+	cost = 10
```

Comment:
```suggestion
	cost = 15
```
15 seems fairer to me personally than 10, as you're paying effectively half of a t1s corpse for a super satchel. Which seems fair to me.

Reply:
I believe no one will realy abuse those packs, because Req got priority on materials, "spec" guns and turrets, if you scared of that because of low price.
15 is one whole mateba, in eyes of RO it may be better not to buy combat pack, which is one mateba, or almost steel sheets.